internal/testrunner/runners/system/runner.go

```diff
@@ -847,6 +851,19 @@ func (r *runner) prepareScenario(ctx context.Context, config *testConfig, svcInf
 		return nil, fmt.Errorf("reading data stream manifest failed: %w", err)
 	}
 
+	// Temporarily until independent Elastic Agents are enabled by default,
+	// enable independent Elastic Agents if package defines that requires root privileges
+	if scenario.pkgManifest.Agent.Privileges.Root || scenario.dataStreamManifest.Agent.Privileges.Root {
```

Comment:
Maybe we need to check if the data stream manifest is nil, for input packages.
```suggestion
	if pkg, ds := scenario.pkgManifest, scenario.dataStreamManifest; pkg.Agent.Privileges.Root || (ds != nil && ds.Agent.Privileges.Root) {
```

Reply:
Sure, better to be sure that it will not raise any error/panic :+1: